edgexfoundry/device-snmp-go
# refactor!: Change configuration and device toml files to yaml

BREAKING CHANGE: Configuration and device files now use yaml

closes:#278







**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/device-snmp-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  <link to docs PR>

## Testing Instructions
1. start edgex with no-secty
2. `run ./device-snmp -cp `
3. verify that configuration and device profiles has loaded
4.  `run ./device-snmp -cp -o`
5.  verify that configuration and device profiles has loaded

## New Dependency Instructions (If applicable)
